### DISCUSSION TITLE
Preguntas bot-whatsapp

### DISCUSSION THREAD

**Discussion by Toninobandolero** [codigoencasa/builderbot#115] [Q&A] [answered]
Como solucionar el Error: listen EADDRINUSE: address already in use :::3000
Que archivos debo tocar para configurar el proyecto con dialogflow?
Que debo hacer para parar la app correctamente desde el terminal? 
Debo borrar algún archivo de sesión/caché/cookies?
Que debo hacer para reiniciar la app correctamente desde el terminal?
Como debo configurar los archivos para ejecutarlo en un VPS?
Que pasa si el VPS se reinicia?

**Comment by pekarnick** [comment, marked as answer]
Paso tiempo, pero si aun no lo solucionaste tenes que cambiar el puerto en archivo .env por otro que no sea el 3000.
